app/wizards/claims/add_claim_wizard/mentor_step.rb

```diff
@@ -0,0 +1,21 @@
+class Claims::AddClaimWizard::MentorStep < BaseStep
+  attribute :mentor_ids, default: []
+
+  validates :mentor_ids, presence: true, inclusion: { in: ->(step) { step.mentors_with_claimable_hours.unscoped.ids } }
```

Comment:
Without `unscoped` you get the following error:
`ActiveRecord::StatementInvalid: PG::InvalidColumnReference: ERROR:  for SELECT DISTINCT, ORDER BY expressions must appear in select list`
This is because ` Claims::MentorsWithRemainingClaimableHoursQuery` returns an ordered collection and `ids` does a `select` which doesn't include the `order` condition. `unscoped` just removes the ordering.